Content.Server/SimpleStation14/Commands/GhostRespawnCommand.cs

```diff
@@ -0,0 +1,63 @@
+using Content.Server.GameTicking;
+using Content.Server.Ghost.Components;
+using Content.Shared.Administration;
+using Content.Shared.CCVar;
+using Content.Shared.SimpleStation14.CCVar;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Console;
+using Robust.Shared.Timing;
+
+namespace Content.Server.SimpleStation14.Commands;
+
+[AnyCommand()]
+public sealed class GhostRespawnCommand : IConsoleCommand
+{
+    [Dependency] private readonly IGameTiming _gameTiming = default!;
+    [Dependency] private readonly IEntityManager _entityManager = default!;
+    [Dependency] private readonly IConfigurationManager _configurationManager = default!;
+
+    public string Command => "ghostrespawn";
+    public string Description => "Allows the player to return to the lobby if they've been dead long enough, allowing re-entering the round AS ANOTHER CHARACTER.";
+    public string Help => $"{Command}";
+
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (!_configurationManager.GetCVar(SimpleStationCCVars.RespawnEnabled))
+        {
+            shell.WriteLine("Respawning is disabled, ask an admin to respawn you.");
```

Comment:
```suggestion
            shell.WriteLine("Respawning is disabled, ask an admin to respawn you if the server allows it.");
```
Maybe not the best thing to put, since some servers might not allow that :P